kahenya-anita/Bumble-Clone
# :recycle: Refactor code to use the correct BEM

## Description

## Ticket[]()
## Types Of Changes
[x] Bug fix

[] New Feature

[] Breaking Changes

## Checklist:
[x]My code follows the code style of this project

[x]All new and existing tests passed.

[x]My code has been linted and formatted correctly.

[x]I have commented my code, particularly in hard-to-understand areas.

[x]My changes generate no new warnings.

## How can this code be tested?
1. After Cloning this repo , run `git pull
` to ensure you are up to date. 
2. Checkout to [Footer_Anita-Kahenya](Footer_Anita-Kahenya) 
3. Run the changes in the browser

![Screenshot 2023-08-21 at 11 26 49 PM](https://github.com/kahenya-anita/Bumble-Clone/assets/62019551/ca4680ee-ee51-483f-bf63-0b143c23446e)
